superset/sql_parse.py

```diff
@@ -35,7 +35,10 @@ def __init__(self, sql_statement):
         self._table_names = set()
         self._alias_names = set()
         self._limit = None
-
+        # Start Dremio Support
+        self.sql = self.sql.replace('[','"')
```

Comment:
A policy we have is that all database-engine-specific logic should live in the `db_engine_specs` module and not be scattered across the codebase. Maybe there's a need to add a new method ` to `db_engine_specs.BaseSpec` that does nothing and override it in `DremioSpec`